CONTRIBUTING.md

```diff
@@ -243,18 +243,87 @@ If in doubt, you can always ask for guidance in the Pull Request or on
+In order to get landed, a Pull Request needs to be reviewed and
+[approved](#getting-approvals-for-your-pull-request) by
+at least one Node.js Collaborator. If the Pull Request touches
+more than the documentation, then it also need to pass a
+[CI (Continuous Integration) test run](#ci-testing). After that,
+as long as there are no objections from a Collaborator, the
+Pull Request can be merged. If you find your Pull Request waiting
+longer than you expect, see the
+[notes about the waiting time](#waiting-until-the-pull-request-gets-landed).
+
+When a collaborator lands your Pull Request, they will post
+a comment to the Pull Request page mentioning the commit(s) it
+landed as. GitHub often shows the Pull Request as `Closed` at this
+point, but don't worry. If you look at the branch you raised your
+Pull Request against (probably `master`), you should see a commit with
+your name on it. Congratulations and thanks for your contribution!
+
+## Additional Notes
+
+### Commit Squashing
+
+When the commits in your Pull Request get landed, they will be squashed
+into one commit per logical change, with metadata added to the commit
+message (including links to the Pull Request, links to relevant issues,
+and the names of the reviewers). As long as you don't delete
+your fork branch, the commit history of your Pull Request will stay
+intact on the Pull Request page.
+
+For the size of "one logical change",
+[0b5191f](https://github.com/nodejs/node/commit/0b5191f15d0f311c804d542b67e2e922d98834f8)
+can be a good example. It touches the implementation, the documentation,
+and the tests, but is still one logical change.
+
+### Getting Approvals for Your Pull Request
+
+A Pull Request is approved either by saying LGTM, which stands for
+"Looks Good To Me", or by using GitHub's Approve button. 
+GitHub's Pull Request review feature can be used during the process.
+For more information, check out
+[the video tutorial](https://www.youtube.com/watch?v=HW0RPaJqm4g)
+or [the official documentation](https://help.github.com/articles/reviewing-changes-in-pull-requests/).
+
+After you push new changes to your branch, you need to get
+approval for these new changes again, even if GitHub shows "Approved"
+because the reviewers have hit the buttons before.
+
+### CI Testing
+
+If your Pull Request contains executable code, it needs to be tested
+to make sure the code works on the platforms that Node.js
+supports. This is done by running the code through the CI system.
+
+Only a Collaborator can request a CI run. Usually one of them will do it
+for you as approvals for the Pull Request come in.
+If not, you can ask a Collaborator to request a CI run.
+
+### Waiting Until the Pull Request Gets Landed
+
+After the Pull Request gets approved and passes the CI,
+it still needs to wait for at least another 48 hours (72
+hours on a weekend) so that everyone has a chance to 
+weigh in. If the changes are trivial, collaborators may 
```

Comment:
I don't think it is is quite right.  It has to wait 48 to 72 hours from when it is submitted not from when it passed the CI or is approved.  I think if the PR has been around for that long and there are only minor changes addressing comments/requested changes then it can land.  Specifically for the CI run its better to land soon after the CI run so that there is less change that interaction with some other change can cause a failure when it lands.